src/main/java/com/rakovets/course/java/core/practice/oop/principles/Cat.java

```diff
@@ -0,0 +1,29 @@
+package com.rakovets.course.java.core.practice.oop.principles;
+
+public class Cat {
+    private String name;
+
+    public Cat(String name) {
+        this.name = name;
+    }
+    public String mew() {
+        return "Mew";
+    }
+    public String purr() {
+        return "Purr";
+    }
+     public String getName() {
+         return this.name;
+     }
+     public void setName(String name) {
+        this.name = name;
+    }
+    public String mew(Person user) {
+        user.changeHappiness(-20);
+        return "Mew";
+    }
+    public String purr(Person user) {
+        user.changeHappiness(20);
+        return "Purr";
+    }
+}
```

Comment:
You have many violations of Code Convention.

```suggestion
public class Cat {
    private String name;

    public Cat(String name) {
        this.name = name;
    }

    public String mew() {
        return "Mew";
    }

    public String purr() {
        return "Purr";
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String mew(Person user) {
        user.changeHappiness(-20);
        return "Mew";
    }

    public String purr(Person user) {
        user.changeHappiness(20);
        return "Purr";
    }
}
```